compiler/rustc_borrowck/src/diagnostics/region_errors.rs

```diff
@@ -421,17 +421,26 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
 
         diag.span_label(*span, message);
 
-        // FIXME(project-rfc-2229#48): This should store a captured_place not a hir id
```

Comment:
Should we keep this comment?

Reply:
no, I think it is addressed by using the field actually